src/main/java/org/codeforamerica/messaging/validators/MessageValidator.java

```diff
@@ -9,9 +9,11 @@ public class MessageValidator implements ConstraintValidator<ValidMessage, Messa
 
     @Override
     public boolean isValid(Message message, ConstraintValidatorContext constraintValidatorContext) {
```

Comment:
Should a message with a toEmail, no subject and a toPhone be considered valid since it can still receive a text?

Reply:
We debated this and decided that having one present indicates intent to send an email, so the sender is likely to resend the request after fixing the validation error. We assume they would prefer to resend the previous request rather than edit it to remove the phone number to prevent duplication.

Reply:
Of course, the presence of toEmail or subject could have been erroneous and it could be seen as frustrating that the text wasn't sent due to an unrelated validation error, so it's not perfect. We will be having conversations with Mary and Platform team devs in the future to work out details like this.

Reply:
If a toEmail, subject and toPhone are _all_ provided, will both an email and a text be sent? Given the current structure of this method, it looks like "validity" prioritizes email over sms and perhaps only an email will be sent - should it be a validation error if all three are present?

Reply:
Yes, both an email and text will be sent. The validity checks that the correct info is present to send either or both, but there's not priority for one over the other.